APITestingFramework/StepDefinitions/Common.cs

```diff
@@ -0,0 +1,26 @@
+﻿using TechTalk.SpecFlow;
+
+namespace Features.GeneralSteps
+{
+    public class CommonSteps
+    {
+        private readonly ScenarioContext _scenarioContext;
+
+        public CommonSteps(ScenarioContext scenarioContext)
+        {
+            _scenarioContext = scenarioContext;
+        }
+
+        [Given(@"the user is authenticated")]
+        public void Giventheuserisauthenticated()
+        {
+            _scenarioContext["Authorization"] = "Basic amdpb2ZmcmVAaG90bWFpbC5jb206UGFzc3dvcmQ=";
```

Comment:
this information should be in a config file

Reply:
Done